mixmatch/config.py

```diff
@@ -28,6 +28,10 @@
                            title='Proxy Config Group')
 
 proxy_opts = [
+    cfg.StrOpt('host',
+               default='http://localhost',
```

Comment:
I think we should construct URLs using the header of the request---that way we can tell if they use the public URL or the private URL or something else